src/js/puzzles.js

```diff
@@ -148,7 +148,7 @@ var puzzlesArray = [
   },
   {
     id: 22,
-    sentence: 'Every rose has its scorns.',
+    sentence: 'Every rose has it‘s scorns.',
```

Comment:
`its` is correct. `it's` is a contraction for `it is`.

Reply:
Whoops, sorry about that.